docs/csharp/events-overview.md

```diff
@@ -65,25 +65,25 @@ an extension of the syntax for delegates.
 To define an event you use the `event` keyword:
 
 ```cs
-public event EventHandler<FileListArgs> OnProgress;
+public event EventHandler<FileListArgs> Progress;
 ```
 
 The type of the event (`EventHandler<FileListArgs>` in this example) must be a
 delegate type. There are a number of conventions that you should follow
 when declaring an event. Typically, the event delegate type has a void return.
-Prefix event declarations with 'On'.
-The remainder of the name is a verb. Use past tense (as in this example) when
+Event declarations should be a verb, or verb phrase.
+Use past tense (as in this example) when
 the event reports something that has happened. Use a present tense verb (for
-example, `OnClosing`) to report something that is about to happen. Often, using
+example, `Closing`) to report something that is about to happen. Often, using
```

Comment:
On the present participle: I'd prefer not going into that much grammatical detail for developers. Would you agree?
On the guidance: I'll do a bit of research and add a paragraph.